src/components/player/atoms/settings/CaptionSettingsView.tsx

```diff
@@ -214,7 +214,7 @@ export function CaptionSetting(props: {
   );
 }
 
-export const colors = ["#ffffff", "#80b1fa", "#e2e535"];
+export const colors = ["#a8a8a8", "#ffffff", "#80b1fa", "#e2e535"];
```

Comment:
```suggestion
export const colors = ["#b0b0b0", "#ffffff", "#80b1fa", "#e2e535"];
```

I have tested b0b0b0 and it works as lowercase, but not as uppercase!

Reply:
Ah alright, I'll use that then. I wonder why uppercase doesn't work?